moebot_bot/util/gifmaker.go

```diff
@@ -0,0 +1,119 @@
+package util
+
+import (
+	"bytes"
+	"image"
+	"image/color"
+	"image/gif"
+	"strings"
+
+	"golang.org/x/image/font"
+	"golang.org/x/image/font/basicfont"
+	"golang.org/x/image/math/fixed"
+)
+
+const lineSpace int = 16
+const xBorder int = 30
+const yBorder int = 14
+const maxWidth int = 500
+
+func MakeGif(text string) []byte {
+	const defaultText string = "Hover to view"
```

Comment:
This and the consts above don't need the `string` and `int`. Go will infer this from the right side (unless you don't want the default types). I'd say it'd be good to remove these types